internal/database/sqlcommon/sqlcommon.go

```diff
@@ -195,7 +177,7 @@ func (s *SQLCommon) queryTx(ctx context.Context, tx *txWrapper, q sq.SelectBuild
 	if err != nil {
 		return nil, tx, i18n.WrapError(ctx, err, coremsgs.MsgDBQueryBuildFailed)
 	}
-	l.Debugf(`SQL-> query: %s`, shortenSQL(sqlQuery))
+	l.Debugf(`SQL-> query %s`, table)
 	l.Tracef(`SQL-> query args: %+v`, args)
```

Comment:
We logs the args at trace level... but shouldn't we log the full SQL statement as well?

Reply:
Thanks for catching that 👍